packages/chakra-ui/test/Form.test.tsx

```diff
@@ -0,0 +1,214 @@
+    expect(tree).toMatchSnapshot();
+  });
+  test("unsupported field", () => {
+    const schema: JSONSchema7 = {
+      type: undefined,
+    };
+    const tree = renderer.create(<Form schema={schema} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("format color", () => {
+    const schema: JSONSchema7 = {
+      type: "string",
+      format: "color",
+    };
+    const tree = renderer.create(<Form schema={schema} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("format date", () => {
+    const schema: JSONSchema7 = {
+      type: "string",
+      format: "date",
+    };
+    const tree = renderer.create(<Form schema={schema} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("format datetime", () => {
+    const schema: JSONSchema7 = {
+      type: "string",
+      format: "datetime",
+    };
+    const tree = renderer.create(<Form schema={schema} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("password field", () => {
+    const schema: JSONSchema7 = {
+      type: "string",
+    };
+    const uiSchema: UiSchema = {
+      "ui:widget": "password",
+    };
+    const tree = renderer
+      .create(<Form schema={schema} uiSchema={uiSchema} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("textarea field", () => {
+    const schema: JSONSchema7 = {
+      type: "string",
+    };
+    const uiSchema: UiSchema = {
+      "ui:widget": "textarea",
+    };
+    const tree = renderer
+      .create(<Form schema={schema} uiSchema={uiSchema} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("select field", () => {
+    const schema: JSONSchema7 = {
+      type: "string",
+      enum: ["foo", "bar"],
+    };
+    const tree = renderer.create(<Form schema={schema} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("checkbox field", () => {
+    const schema: JSONSchema7 = {
+      type: "boolean",
+    };
+    const tree = renderer.create(<Form schema={schema} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("checkbox field", () => {
+    const schema: JSONSchema7 = {
+      type: "boolean",
+    };
+    const tree = renderer.create(<Form schema={schema} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("checkboxes field", () => {
+    const schema: JSONSchema7 = {
+      type: "array",
+      items: {
+        type: "string",
+        enum: ["foo", "bar", "fuzz", "qux"],
+      },
+      uniqueItems: true,
+    };
+    const uiSchema: UiSchema = {
+      "ui:widget": "checkboxes",
+    };
+    const tree = renderer
+      .create(<Form schema={schema} uiSchema={uiSchema} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("radio field", () => {
+    const schema: JSONSchema7 = {
+      type: "boolean",
+    };
+    const uiSchema: UiSchema = {
+      "ui:widget": "radio",
+    };
+    const tree = renderer
```

Comment:
Maybe we should remove this snapshot for now. It looks like the `name` field is generated non-deterministically, so the snapshots can't stay the same:

![image](https://user-images.githubusercontent.com/1689183/111927492-4b17c880-8a87-11eb-8387-921c03dd7047.png)

This is what's currently causing the CI tests to fail.